packages/components/src/templates/dropdown-with-navigation-items.stories.ts

```diff
@@ -0,0 +1,51 @@
+import '../solid-components';
+import { html } from 'lit-html';
+
+export default {
+  tags: ['!dev'],
+  title: 'Templates/Dropdown with navigation items',
+  parameters: {
+    chromatic: { disableSnapshot: true }
+  }
+};
+
+/**
+ *
+ * This is an example of a dropdown.
+ *
+ * ```
+ * ```
+ */
+export const Default = {
+  render: () => html`
+    <style>
+      #anchor--templates-dropdown-with-navigation-items--default .innerZoomElementWrapper {
+        min-height: 500px;
+      }
+    </style>
+
+    <sd-dropdown open>
+      <sd-navigation-item slot="trigger" vertical>
+        <sd-icon name="system/globe" class="h-6 w-6"></sd-icon>
+      </sd-navigation-item>
+      <div class="flex flex-col p-4">
```

Comment:
Slot padding should be:
![Bildschirmfoto 2024-08-31 um 22 16 14](https://github.com/user-attachments/assets/79498e52-251a-46f1-a9d2-1c47d8f86faa)

Reply:
Updated